### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by djmaisen** [pjz9n-pm-pl/Mission#4]
Internal Server Error when i use /mission

**Comment by PJZ9n** [comment]
Would you please send the error message with the stack trace?

Example:
```
[00:00:00] [Server thread/CRITICAL]: pjz9n\mission\exception\NotFoundException: "Progresses for "foo" not found" (EXCEPTION) in "path/to/plugin/src/pjz9n/mission/mission/progress/ProgressList" at line 44
↓
[00:00:00] [Server thread/CRITICAL]: #0 plugins/Mission/src/pjz9n/mission/Main(109): pjz9n\mission\mission\progress\ProgressList::getAll(string[3] foo)
[00:00:00] [Server thread/CRITICAL]: #1 src/pocketmine/plugin/PluginBase(116): pjz9n\mission\Main->onEnable()
[00:00:00] [Server thread/CRITICAL]: #2 src/pocketmine/plugin/PluginManager(552): pocketmine\plugin\PluginBase->setEnabled(boolean 1)
[00:00:00] [Server thread/CRITICAL]: #3 src/pocketmine/Server(1787): pocketmine\plugin\PluginManager->enablePlugin(object pjz9n\mission\Main)
[00:00:00] [Server thread/CRITICAL]: #4 src/pocketmine/Server(1773): pocketmine\Server->enablePlugin(object pjz9n\mission\Main)
[00:00:00] [Server thread/CRITICAL]: #5 src/pocketmine/Server(1586): pocketmine\Server->enablePlugins(integer 1)
[00:00:00] [Server thread/CRITICAL]: #6 src/pocketmine/PocketMine(273): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[16] /path/to/server/root/, string[24] /path/to/plugins/)
[00:00:00] [Server thread/CRITICAL]: #7 src/pocketmine/PocketMine(304): pocketmine\server()
[00:00:00] [Server thread/CRITICAL]: #8 (11): require(string[64] phar://path/to/server/src/pocketmine/PocketMine.php)
↑
```

**Comment by PJZ9n** [comment]
Perhaps this has been fixed since the [1.0.2 Release](https://github.com/pjz9n-pm-pl/Mission/releases/tag/1.0.2). If you continue to have problems, reopen it.
